Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -283,8 +284,9 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
                 method.setAttrs({
                     'multiple': this.get('multiple'),
                     'loadContent': this.get('loadContent'),
+                    'startingLocationId': startingLocationId,
                     'visible': visible,
-                    'isSelectable': Y.bind(this.get('isSelectable'), this)
+                    'isSelectable': Y.bind(this.get('isSelectable'), this),
```

Comment:
Nitpick: not needed :) and funny, as we add the coma in order to have useless diff